### PR TITLE
Add in Driver and a bit more

### PR DESCRIPTION
**Pull request by profxj**
As titled.

### CODE REVIEW

File: frb/associate/chance.py [hunk 1]
```diff
@@ -6,15 +6,38 @@
...
     resource_filename('frb', os.path.join('data', 'Galaxies', 'driver2016_t3data.fits')))
 eb17_spl = hosts.interpolate.UnivariateSpline(x=mag_uniq,
                                    y=np.log10(r_dat),
                                    bbox=[-100, 100],
                                    k=3)
-def n_gal(m_r):
-    return 10 ** eb17_spl(m_r)
+'''
+
+# Spline parameters(globals) are for rmag vs sigma
```
Comment (tarraneh) [inline frb/associate/chance.py:19]: `#Spline parameters (globals) are for m_r vs sigma`

File: frb/associate/chance.py [hunk 2]
```diff
@@ -6,15 +6,38 @@
...
+driver_tck = (np.array([15., 15., 15., 15., 30., 30., 30., 30.]),
+       np.array([-6.41580144, -3.53188049, -1.68500105, -0.63090954, 0., 0., 0., 0.]), 3)
+driver_spl = hosts.interpolate.UnivariateSpline._from_tck(driver_tck)
+
+
+
+def driver_sigma(m_r):
```
Comment (tarraneh) [inline frb/associate/chance.py:26]: `def driver_sigma(rmag)`

File: frb/associate/chance.py [hunk 3]
```diff
@@ -6,15 +6,38 @@
...
+    """
+    Estimated incidence of galaxies per sq arcsec with r > rmag 
+    using Driver et al. 2016 number counts.
+
+    Spline parameters (globals) are for rmag vs sigma
+
+    Args:
+        m_r (float or np.ndarray):
```
Comment (tarraneh) [inline frb/associate/chance.py:34]: `rmag (float or np.ndarray)`

File: frb/associate/chance.py [hunk 4]
```diff
@@ -6,15 +6,38 @@
...
+
+    Returns:
+        float or np.ndarray:  Galaxy number density
+
+    """
+    return 10**driver_spl(m_r)
```
Comment (tarraneh) [inline frb/associate/chance.py:40]: `return 10**driver_spl(rmag)`

**Review by tarraneh — CHANGES_REQUESTED** [review]
Minor change to `driver_sigma()` function: use `rmag` instead of `m_r` for consistency.

File: frb/associate/chance.py (hunk 2 above)
Comment (profxj) [reply frb/associate/chance.py:26]: modified.

File: frb/associate/chance.py (hunk 3 above)
Comment (profxj) [reply frb/associate/chance.py:34]: same

File: frb/associate/chance.py (hunk 4 above)
Comment (profxj) [reply frb/associate/chance.py:40]: done

**Comment by profxj** [comment]
merging.

**Pull request merged**